sky/clouds/kubernetes.py

```diff
@@ -689,6 +689,11 @@ def _check_compute_credentials(cls) -> Tuple[bool, Optional[str]]:
             except Exception as e:  # pylint: disable=broad-except
                 return (False, f'Credential check failed for {context}: '
                         f'{common_utils.format_exception(e)}')
+            unlabeled_nodes = kubernetes_utils.get_unlabeled_accelerator_nodes(
+                context)
+            if len(unlabeled_nodes) > 0:
+                hints.append(f'Context {context} has {len(unlabeled_nodes)} '
+                             f'unlabeled nodes with accelerators.')
```

Comment:
This is awesome @SeungjinYang!

We should always add an action the user should follow to fix the situation, e.g. `run: python -m gpu_labeler`